shared/domains/chelonia/internals.js

```diff
@@ -1149,12 +1149,12 @@ export default (sbp('sbp/selectors/register', {
                     // reason to remain subscribed to this contract. In this
                     // case, attempt to release it.
                     sbp('chelonia/contract/release', foreignContract, { try: true }).catch(e => {
-                      console.error(`[chelonia] Error at OP_KEY_DEL internalSideEffectStack while attempting to release foreign contract ${foreignContract}`, e)
+                      logEvtError(message, `[chelonia] Error at OP_KEY_DEL internalSideEffectStack while attempting to release foreign contract ${foreignContract}`, e)
```

Comment:
Is this really something that can happen with an outgoing message that is OK to leave as a warning?

Reply:
Outgoing messages typically don't have side effects and the `internalSideEffectStack` isn't used. So, this is mostly a discussion about hypotheticals.

As for your question, if we ever did execute side effects for outgoing messages, I presume it'll be in some kind of sandboxed way and errors still wouldn't have the same way. Since this isn't what happens and I can't think of good reasons to run side effects for outgoing messages, nor a way to do so cleanly, I'm just guessing here.

Reply:
If this shouldn't ever happen for outgoing messages, then perhaps this should be left as `console.error`, no? As this happening should be an exceptional circumstance that we should take note of if it does happen (since you say it shouldn't). If my reasoning sounds OK to you, then please revert.

Reply:
> then perhaps this should be left as console.error, no?

Maybe, I see arguments both ways. Happy to revert though. However,

> that we should take note of if it does happen (since you say it shouldn't)

If this is the reason, I don't think we should rely on obscure errors happening to detect this. If we want truly defensive coding, maybe at the top of this side effect there could be `if (outgoing) throw Error('Something really bad just happened')`.

Reasons for leaving it as is: 
 * Consistency
Reasons for reverting:
 * Currently, if it happened it'd likely be an error

Should I revert?

Reply:
> If this is the reason, I don't think we should rely on obscure errors happening to detect this. If we want truly defensive coding, maybe at the top of this side effect there could be if (outgoing) throw Error('Something really bad just happened').

Sure, feel free to do that, then I would also revert as well, since it doesn't make sense to have `logEvtError` there in that case either because `logEvtError` implies it's a piece of code that can run on outgoing messages (and needs to be logged as a warning in that case).